app/helpers/handlers.js

```diff
@@ -489,6 +491,15 @@ const getPage = async (question, request, h) => {
   }
 
   if (url === 'project-cost') {
```

Comment:
all of these setYarValue fields can instead be reset in the questionBank. See **yarKeysReset** usage for examples